Documentation/GeneralConventions/FileStructure.rst

```diff
@@ -0,0 +1,810 @@
+details.
+
+.. note::
+
+   You can reduce the single file documentation even further by omitting the
+   Settings.cfg - which is even less recommended, but works. All Sphinx
+   theme variables will then fall back to their default values, e.g. the project
+   title will then be "The Project's Title" and the release version "0.0.0".
+
+
+.. _examples:
+
+Examples
+========
+
+These are some good examples of TYPO3 full and single file documentation in the
+wild.
+
+
+.. _examples-official-typo3-manuals:
+
+Official TYPO3 manuals
+----------------------
+
+.. table::
+   :widths: 25, 75
+
+   =========================  ==================================================
+   Project                    Links
+   =========================  ==================================================
+   TSconfig Reference (full)  `README.md <https://github.com/TYPO3-Documentation/TYPO3CMS-Reference-TSconfig>`__ |
+                              `Settings.cfg <https://github.com/TYPO3-Documentation/TYPO3CMS-Reference-TSconfig/blob/main/Documentation/Settings.cfg>`__ |
+                              `Index.rst <https://github.com/TYPO3-Documentation/TYPO3CMS-Reference-TSconfig/blob/main/Documentation/Index.rst>`__ |
+                              `Includes.rst.txt <https://github.com/TYPO3-Documentation/TYPO3CMS-Reference-TSconfig/blob/main/Documentation/Includes.rst.txt>`__ |
+                              `Read online <https://docs.typo3.org/m/typo3/reference-tsconfig/main/en-us/>`__
+   =========================  ==================================================
+
+
+.. _examples-typo3-system-extension-documentation:
+
+TYPO3 system extension documentation
+------------------------------------
+
+.. table::
+   :widths: 25, 75
+
+   ===========================  ================================================
+   Project                      Links
+   ===========================  ================================================
+   Import / Export (full)       `README.rst <https://github.com/TYPO3/typo3/tree/main/typo3/sysext/impexp>`__ |
+                                `Settings.cfg <https://github.com/TYPO3/typo3/blob/main/typo3/sysext/impexp/Documentation/Settings.cfg>`__ |
+                                `Index.rst <https://github.com/TYPO3/typo3/blob/main/typo3/sysext/impexp/Documentation/Index.rst>`__ |
+                                `Includes.rst.txt <https://github.com/TYPO3/typo3/blob/main/typo3/sysext/impexp/Documentation/Includes.rst.txt>`__ |
+                                `Read online <https://docs.typo3.org/c/typo3/cms-impexp/main/en-us/>`__
+   ===========================  ================================================
+
+
+.. _examples-typo3-third-party-extension-documentation:
+
+TYPO3 third-party extension documentation
+-----------------------------------------
+
+.. table::
+   :widths: 25, 75
+
+   ========================  ===================================================
```

Comment:
I think we should get rid of these tables and not introduce new ones. They break easily and do not display well on smaller screens. We could convey the same information with definition lists or headlines or some such

Reply:
They probably break for more columns or for longer words. For two columns and short words they work fine:

![Screen Shot 2022-03-14 at 15 55 16](https://user-images.githubusercontent.com/20297232/158198999-739f4b2b-6c7c-4b75-8b84-f0d8210d96bd.png)

Reply:
The information we provide here isn't really necessarily  of tabular nature so I would not use tables for it.

Reply:
So this is about taste. Which content element would you prefer?

Reply:
I would probably have solved it with a definition List or another List of some sorts. If I would introduce a table I would prefer one of the easier extendable formats like https://docs.typo3.org/m/typo3/docs-how-to-document/main/en-us/WritingReST/Tables.html#csv-table however, the discussion on weather we get rid of the first two table formats is out of scope here. So I am ok with your tables for now, we can always change them later on.